dependencies.props

```diff
@@ -10,40 +10,37 @@
   -->
   <PropertyGroup>
     <WCFCurrentRef>bc189e7681f4e64bb2ba779eefb40b9e4542c683</WCFCurrentRef>
+    <CoreFxCurrentRef>71db6a73ef6b8781f0c05372eca442fd0d72ef7c</CoreFxCurrentRef>
+    <CoreClrCurrentRef>71db6a73ef6b8781f0c05372eca442fd0d72ef7c</CoreClrCurrentRef>
+    <CoreSetupCurrentRef>71db6a73ef6b8781f0c05372eca442fd0d72ef7c</CoreSetupCurrentRef>
     <StandardCurrentRef>c520a2569b40fc53cf51e4f6970c3e7411adc173</StandardCurrentRef>
     <BuildToolsCurrentRef>8fc22d261883eb7a1f8765bfe716d806303d0318</BuildToolsCurrentRef>
   </PropertyGroup>
 
-  <!-- Auto-upgraded properties for each build info dependency. -->
+  <!-- Product dependency versions. -->
   <PropertyGroup>
-    <WCFExpectedPrerelease>preview3-26429-01</WCFExpectedPrerelease>
-    <CoreFxExpectedPrerelease>preview3-26430-04</CoreFxExpectedPrerelease>
     <NETStandardPackageVersion>2.0.3</NETStandardPackageVersion>
     <NETStandardPackageId>NETStandard.Library</NETStandardPackageId>
-    <!--
-      This property is used for many CoreFX packages, because they have the same
-      versions. Make more properties for CoreFX if split versions are required.
-    -->
+    <CoreFxProductDependencyVersion>4.5.0</CoreFxProductDependencyVersion>
```

Comment:
I think it should be pointed out that this version isn't auto-updated.

Also, since this isn't auto-updated, and it's just one version for all the CoreFX packages, the removed comment above actually applies to this situation.

If there are only two dependencies using this (using find only shows me two, at least) then making an auto-upgraded property for each individual package would actually be good IMO for future servicing.

Reply:
The reason for not auto-updating those two CoreFx product dependencies is because we don't want to update those two dependencies unless absolutely necessary. So even if there are newer versions of those packages we don't want our released WCF packages forcing an upgrade of those two packages unless we have some specific reason for doing so.

However, I would like our tests to run against the latest version of these two dependencies and I was going to ask you about how to do that.

Reply:
Ah, that makes sense. I'd do something like this:

```xml
<!--
  Summary of the description you just gave. 😄 Plus the note
  about splitting CoreFxProductDependencyVersion if needed.
-->
<SystemReflectionDispatchProxyPackageVersion>4.5.0</SystemReflectionDispatchProxyPackageVersion>
<SystemNetHttpWinHttpHandlerPackageVersion>4.5.0</SystemNetHttpWinHttpHandlerPackageVersion>

... (in the tests section) ...
<SystemReflectionDispatchProxyTestPackageVersion>4.5.1 (or whatever)</SystemReflectionDispatchProxyTestPackageVersion>
<SystemNetHttpWinHttpHandlerTestPackageVersion>4.5.2</SystemNetHttpWinHttpHandlerTestPackageVersion>
```

Then hook up the `TestPackageVersion` props to auto-update and have those in the test runtime, while using `CoreFxProductDependencyVersion` to compile against/depend on.

(Putting the updated dependencies in the test runtime is the part I don't know as much about. It sounds right but I'm not completely sure how that works out when running the tests.)